store/kv/chunk.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2022-2023 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/tigrisdata/tigris/internal"
+	"github.com/tigrisdata/tigris/server/config"
+)
+
+const (
+	KB   = 1000
+	KB99 = 99 * KB
+
+	chunkIdentifier = "__"
```

Comment:
maybe something more human friendly, like `__chunk_`

Reply:
It will be added to all the chunked keys, better to optimize it on what we are adding.